src/compo/gcas_nc2ioda.py

```diff
@@ -0,0 +1,316 @@
+            except IOError:
+                raise IOError('%s file not found!' % self.filename)
+            except Exception:
+                raise Exception('Unknown error opening %s' % self.filename)
+
+            # Read global attributes
+            self.AttrData['platform'] = dsFlight.attrs['MeasurementPlatform']
+            self.AttrData['description'] = dsFlight.attrs['RevisionNumber']
+
+            # Read lat lon and flatten from 2d (time,xtrack) and make it 1d
+            flats = dsFlight['lat'].values.flatten()
+            flats = np.where(np.isnan(flats), float_missing_value, flats)
+            lats = flats.astype(np.float32)
+
+            flons = dsFlight['lon'].values.flatten()
+            flons = np.where(np.isnan(flons), float_missing_value, flons)
+            lons = flons.astype(np.float32)
+
+            alts_1d = dsFlight['aircraft_altitude'].values.flatten()
+            xtrack = np.arange(dsFlight.dims['xtrack'])  # pixels in swath
+            nlocs = lats.shape[0]
+            alts = np.repeat(alts_1d, len(xtrack))  # repeat alt for each swath
+
+            lats = lats.astype(np.float32)
+            lons = lons.astype(np.float32)
+            alts = alts.astype(np.float32)
+
+            # Read 3d model alt (level) and reshape it to 2d
+            geoscf = xr.open_dataset(filename, group='_Model_Information')
+            # Model alt label in incorrect, change km to m
+            fmodel_alt = geoscf['model_altitude']*1000
+            level_dim, time_dim, xtrack_dim = fmodel_alt.shape
+            model_alt = fmodel_alt.values.reshape(level_dim, time_dim * xtrack_dim)
+
+            # Read 3d model pressure and reshape it to 2d
+            fmodel_press = geoscf['model_pressure']*HPA2PA
+            model_press = fmodel_press.values.reshape(level_dim, time_dim * xtrack_dim)
+            model_press = np.where(np.isnan(model_press), float_missing_value, model_press)
+            model_press = model_press.astype(np.float32)
+
+            # Find aircraft P using model pressure and altitude info
+            aircraft_p = np.zeros(time_dim * xtrack_dim, dtype=np.float32)
+            for t in range(time_dim * xtrack_dim):
+                differences = np.abs(np.array(model_alt[:, t]) - np.array(alts[t]))
+                idx = np.argmin(differences)
+                aircraft_p[t] = model_press[idx, t]
+
+            # Only 2 vertice: surface pressure and flight pressure
+            # For total column flight_pressure is 0
+            num_vert = 2
+            pressure_vertice = np.zeros([time_dim * xtrack_dim, num_vert], dtype=np.float32)
+            pressure_vertice[:, 1] = model_press[level_dim-1, :]  # surface pressure from model
+
+            # Read time and convert to ioda time format
+            obs_time = dsFlight['time'].values
+            time = np.repeat(obs_time, len(xtrack))
+
+            times = (time - np.datetime64(epoch)) / np.timedelta64(1, 's')
```

Comment:
```suggestion
            times = (time - np.datetime64(epoch.strftime("%Y-%m-%dT%H:%M:%S"))) / np.timedelta64(1, 's')
```
Then it can remove the time zone info in epoch for datetime64.

Reply:
yes, this removes the warning message. thanks.